config/locales/en.yml

```diff
@@ -2430,9 +2430,10 @@ en:
           select_all: Select all
           allocated_to: Allocated to
           disk_evidence: Disk evidence
-          no_claims_allocated: No claims need to be allocated
+          no_claims_allocated: There is no claim available for allocation
           number_of_claims: 'Number of claims:'
           choose_label: Choose claim number %{case_number}
+          choose_label_html: <span class="govuk-visually-hidden">Select case %{case_number}</span>
```

Comment:
[ditto](https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/pull/4142/files#r676749436)